app/Services/Phoenix/Phoenix.php

```diff
@@ -69,6 +68,21 @@ private function getAuthenticationCookie()
     }
 
     /**
+     * Send a POST request to save a copy of the reportback in Phoenix.
+     *
+     * @param string $nid
+     * @param array $body
+     * @param bool $withAuthorization - Should this request be authorized?
+     * @return object|false
+     */
+    public function postReportback($nid, $body = [])
+    {
+        $response = $this->post($this->url . 'campaigns/' . $nid . '/reportback', $body, $withAuthorization = true);
```

Comment:
The `$withAuthorization` parameter will default to true, so that can be omitted here.